src/services/auth.services.ts

```diff
@@ -71,9 +74,9 @@ export const login = async (
         message: "Invalid credentials",
       };
     }
-    // validate password
-    const validPassword = await user.validatePassword(data.password);
-    if (!validPassword) {
+    // hash password
+    const validPassword = await compare(data.password, user.password)
```

Comment:
i have changed this stuff already  the validationPassword function does that from the model